### PR TITLE
my commit

### PR DESCRIPTION
**Pull request by himanshu004**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by himanshu004** [comment]
> Hey @himanshu004,
> 
> Congrats on graduating! 🎓 🎉
> 
> Unfortunately, the image is not squared (1:1 or width = height) 🔳 The size of the image is 812x895 (pixels). Could you please make this change? 📝
> 
> Thanks,
> Mohammed Shoaib

> I have made the change you told.Thanks.

**Comment by himanshu004** [comment]
> Hey @himanshu004,
> 
> Thank you for correcting to a square image! However, the image size is now smaller.
> The size of the image must be **at least** 544x544 (pixels), yours is 529x529 (pixels). Could you please make this change? 📝
> 
> Thanks,
> Mohammed Shoaib

>Changed the image size too.

File: _data/himanshu004/himanshu004.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Himanshu Madan
+institution: Deen Dayal Updhyaya College
+profile_pic: me2.jpg 
+quote: Falling into the river does not drowns you, not willing to swim does.
```
Comment (Mohammed-Shoaib) [inline _data/himanshu004/himanshu004.md:5]: Fix grammatical error? `drowns` to `drown`.

Comment (himanshu004) [reply _data/himanshu004/himanshu004.md]: Corrected the error.

**Comment by himanshu004** [comment]
> Hey @himanshu004,
> 
> Thanks for fixing up the image! It seems like there is a grammatical error in your quote. Would you consider fixing it? 📝
> 
> Thanks,
> Mohammed Shoaib

> Fixed it.

**Comment by Mohammed-Shoaib** [comment]
@mitulsavani Thank you for reviewing this PR! :heart:

**Comment by himanshu004** [comment]
> @mitulsavani Thank you for reviewing this PR! ❤️


> Kindly merge the commits.Thank you.

**Comment by mitulsavani** [comment]
> > Kindly merge the commits.Thank you.

You PR will be merged soon by one of the staff members

**Comment by Mohammed-Shoaib** [comment]
Your pull request is perfect! :confetti_ball:

However, I or @mitulsavani cannot merge it as we do not have write access. You'll have to wait for a reviewer with _write_ access to merge your pull request.

Till then sit back, relax, and enjoy :relaxed:

**Comment by himanshu004** [comment]
> @himanshu004 Your pull request is perfect! 🎊
> 
> However, I or @mitulsavani cannot merge it as we do not have write access. You'll have to wait for a reviewer with _write_ access to merge your pull request.
> 
> Till then sit back, relax, and enjoy ☺️

> Okay. Thanks for your prompt reply. Appreciate it.

**Pull request merged**